nilearn/datasets/func.py

```diff
@@ -1276,3 +1276,174 @@ def fetch_mixed_gambles(n_subjects=1, data_dir=None, url=None, resume=True,
+        'ts2' - choice denotes the timeseries signals which were extracted using
+                mutiple spatial regression.
+        'ts3' - choice denoted the timeseries signals which were extracted using
+                eigen regression.
+
+        If chosen to be default, network matrices data estimated using all
+        timeseries extraction method ['ts2', 'ts3'] are fetched.
+
+        If chosen a string, network matrices estimated using particular type
+        of timeseries extraction method is fetched.
+
+    resume: boolean, Default is True
+        This parameter is required if a partly downloaded file is needed to be
+        resumed to download again.
+
+    verbose: int, Default is 1
+        This parameter is used to set the verbosity level to print the message
+        to give information about the processing.
+        0 indicates no information will be given.
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        Dictionary-like object, attributes are:
+
+        'Znet1': Full correlation matrices
+
+        'Znet2': Partial correlation matrices
+
+    References
+    ----------
+    For more details:
+    Stephen Smith et al, HCP beta-release of the Functional Connectivity MegaTrawl.
+    April 2015 "HCP500-MegaTrawl" release.
+
+    https://db.humanconnectome.org/megatrawl/
+
+    Disclaimer
+    ----------
+    IMPORTANT: This is open access data. You must agree to Terms and conditions
+    of using this data before using it,
+    available at: http://humanconnectome.org/data/data-use-terms/open-access.html
+    Open Access Data (all imaging data and most of the behavioral data)
+    is available to those who register an account at ConnectomeDB and agree to
+    the Open Access Data Use Terms. This includes agreement to comply with
+    institutional rules and regulations. This means you may need the approval
+    of your IRB or Ethics Committee to use the data. The released HCP data are
+    not considered de-identified, since certain combinations of HCP Restricted
+    Data (available through a separate process) might allow identification of
+    individuals. Different national, state and local laws may apply and be
+    interpreted differently, so it is important that you consult with your IRB
+    or Ethics Committee before beginning your research. If needed and upon request,
+    the HCP will provide a certificate stating that you have accepted the
+    HCP Open Access Data Use Terms. Please note that everyone who works with
+    HCP open access data must review and agree to these terms, including those
+    who are accessing shared copies of this data. If you are sharing
+    HCP Open Access data, please advise your co-researchers that they must
+    register with ConnectomeDB and agree to these terms.
+    Register and sign the Open Access Data Use Terms at
+    ConnectomeDB: https://db.humanconnectome.org/
+    """
+    # Data is manually uploaded in NITRC
```

Comment:
is -> has been or just remove this comment. I don't think it adds a lot of value.